uat/README.md

```diff
@@ -6,6 +6,17 @@ the `aws.greengrass.clientdevices.Auth` component.
 
 ## Running UATs locally
 
+### Requirements
+Some scenarios require to install and run the EMQX, Auth, IPDetector Greengrass components.
+
+#### Docker
+Because EMQX compoment running in a docker containter docker should be installed on workstation where scenario is running.
+
+#### Rrivileges
```

Comment:
typo

Reply:
Fixed in 9ab68b0a061e41903c9de947922073777de93463